testing/log4cxx/APKBUILD

```diff
@@ -0,0 +1,52 @@
+# Contributor: Russ Webber <russ@rw.id.au>
+# Maintainer: Russ Webber <russ@rw.id.au>
+pkgname=log4cxx
+pkgver=0.10.0
+pkgrel=0
+pkgdesc="A C++ port of the Log4j logging framework."
+url="http://logging.apache.org/log4cxx"
+license="Apache-2.0"
+options="!check"  # testsuite in 0.10.0 does not work on alpine
+subpackages="$pkgname-dev"
```

Comment:
```suggestion
subpackages="$pkgname-static $pkgname-dev"
```

Reply:
What is the static convention used for?

Reply:
https://github.com/alpinelinux/abuild/pull/66

It's to hold the `lib*.a` files.